test/User.js

```diff
@@ -270,4 +273,65 @@ describe('User', () => {
       result.body.should.have.property('message');
     });
   });
+
+  describe('/POST connectToDiscord', () => {
+    it('Should return statusCode 403 if no token was passed in', async () => {
+      const user = {
+        email: 'a@b.c'
+      };
+      const result = await test.sendPostRequest(
+        '/api/user/connectToDiscord', user);
+      expect(result).to.have.status(FORBIDDEN);
+    });
+    it('Should return statusCode 401 if an invalid ' +
+      'token was passed in', async () => {
+      const user = {
+        email: 'a@b.c',
+        token: 'Invalid token'
+      };
+      const result = await test.sendPostRequest(
+        '/api/user/connectToDiscord', user);
+      expect(result).to.have.status(UNAUTHORIZED);
+    });
+    it('Should return statusCode 400 if an incorrect or no ' +
+      'email was used', async () => {
+      const user = {
+        token: token
```

Comment:
```suggestion
        token
```

Reply:
what about for the rest of the tests that are written in this way? those necessary to change as well?

Reply:
changing just these is fine